omnibus.rb

```diff
@@ -24,6 +24,10 @@
 # package_dir   './local/omnibus/pkg'
 # package_tmp   './local/omnibus/pkg-tmp'
 
+# Windows architecture defaults - set to x86 unless otherwise specified.
+# ------------------------------
+windows_arch   ENV['OMNIBUS_WINDOWS_ARCH'] || :x86
```

Comment:
I think we've been using :x86 throughout omnibus software.  I think I'll do a :to_sym and a quick check instead.  The environment variable is to assist eng services.  They can bake the 64-bit windows builders with this environment variable instead of having to patch this file every time they clone.